lib/index.js

```diff
@@ -79,12 +79,6 @@ module.exports = {
         );
       }
 
-      if (optionalFeaturesMissing || optionalFeatures.isFeatureEnabled('jquery-integration')) {
```

Comment:
Instead of removing this, we should ensure that the flag is set (not `undefined`).